docs/source/index.mdx

```diff
@@ -1,26 +1,30 @@
 ---
-title: Welcome
-description: Start here to learn about the Apollo platform
+title: Welcome to the Apollo Docs
```

Comment:
We have other work in progress for renaming the core docset and pulling the Graph Manager docs out into their own docset. This would probably lead to us renaming this article a second time, so if it sounds good to you, I'll revert this particular change for now and we can revisit when the time comes

Reply:
Sounds good to me 👍